### PR TITLE
Show side bias on standings for Single-sided swiss rounds.

### LINKED ISSUE #336: Add side bias to standings for SSS rounds
Aesop's View
Screenshot 2024-09-07 at 12 18 53 PM

Current Cobra view:
Screenshot 2024-09-07 at 12 19 18 PM

### PR DESCRIPTION
**Pull request by plural**
Fixes #336 

Screenshot 2024-09-07 at 4 18 18 PM

### CODE REVIEW

File: app/views/rounds/_pairings.html.slim [hunk 1]
```diff
@@ -72,7 +72,7 @@
               div class='form-group'
                 div class='form-check form-check-inline'
                   = f.check_box :intentional_draw, class: 'form-check-input'
-                  = f.label :intentional_draw, 'Intentional Draw', class: 'form-check-label'
+                  = f.label :intentional_draw, 'ScIntentional Draw', class: 'form-check-label'
```
Comment (haradan) [inline app/views/rounds/_pairings.html.slim:75]: It looks like this was committed accidentally?

Comment (plural) [reply app/views/rounds/_pairings.html.slim]: ha!  My debugging is showing.  :)

File: app/controllers/players_controller.rb [hunk 2]
```diff
@@ -200,7 +201,8 @@ def render_player_list_for_standings(stage)
         extended_sos: 0,
         corp_points: 0,
         runner_points: 0,
-        manual_seed: player.manual_seed
+        manual_seed: player.manual_seed,
+        side_bias: stage.format == 'single_sided_swiss' ? player.side_bias : nil
```
Comment (haradan) [inline app/controllers/players_controller.rb:205]: This method is just for the player meeting, so it won't have side bias. Maybe the method should be renamed?

**Review by haradan — APPROVED** [review]
Looks good :)

File: app/controllers/players_controller.rb (hunk 2 above)
Comment (plural) [reply app/controllers/players_controller.rb]: Ah, good call.  Done.

**Pull request merged**